CRM/Financial/Form/PaymentProcessorFormTrait.php

```diff
@@ -0,0 +1,42 @@
+<?php
```

Comment:
@eileenmcnaughton just a question how does this trait relate to the PR?

Reply:
@seamuslee001 mis-commit

Reply:
gone